src/ADAL.PCL.Android/Properties/AssemblyInfo.cs

```diff
@@ -39,6 +39,3 @@
 [assembly: Guid("ff47962a-d498-4c63-b7e9-4db3653ad7df")]
 
 // Assembly version information is in file ADAL.Common\CommonAssemblyInfo.cs
-
-[assembly: AssemblyDelaySign(true)]
-[assembly: AssemblyKeyFile(@"..\..\build\272MSSharedLibSN2048.snk")]
```

Comment:
The ADAL.PCL.Android project was the only one that set the signing properties using attributes. I've changed it to use the project settings like all of the other projects.